code/__DEFINES/dcs/signals/signals_object.dm

```diff
@@ -336,6 +336,15 @@
 ///sent to targets during the process_hit proc of projectiles
 #define COMSIG_PELLET_CLOUD_INIT "pellet_cloud_init"
 
+// /obj/vehicle/sealed/car/vim signals
+
+///from /datum/action/vehicle/sealed/noise/chime
+#define COMSIG_VIM_CHIME_USED "vim_chime_used"
+///from /datum/action/vehicle/sealed/noise/buzz
+#define COMSIG_VIM_BUZZ_USED "vim_buzz_used"
+///from /datum/action/vehicle/sealed/headlights/vim
+#define COMSIG_VIM_HEADLIGHTS_TOGGLED "vim_headlights_toggled"
```

Comment:
There are all sent off Trigger right? why not use `COMSIG_ACTION_TRIGGER` instead? it's set anytime an action is triggered

Reply:
Oh..... Nevermind.
Fucking mechcode